### ISSUE TITLE
AOE and dmg multiplier don't work

### ISSUE THREAD

**Issue by MipPim** [tf2td/towerdefense#16]
After having a few issues with the sql part, I managed to have the server running. Everything went well as long as I didn't try to summon any entity like a custom sentry, ammo packs, etc via the console, but then when after wave 22 on firstone (the other maps won't work for some reasons) nothing happen when I shoot or hit the buttons to upgrade damage multipliers. I can spawn and upgrade bot turrets without any issues but the multipliers don't work.
Another thing is AOE bot turrets don't work either, and I've seen that there already is a resolved issue about that, but no solution seems to be mentioned anywhere ?
TL/DR: Damage multipliers and AOE turrets don't work, so I hope I can get some help please ! Thank you in advance.

**Comment by Dragonisser** [comment]
Possible a map issue and code issue. Needs to be debugged.

**Comment by Yugboon** [comment]
Having the same issue. Multipliers don't work on any map.

**Comment by Dragonisser** [comment]
AOE Engi works as expected. It doesnt seem like its suppossed to upgrade towers/buildings but rather just filling sentries.

Multiplier dont work because of missing database entries https://github.com/tf2td/towerdefense/issues/28

**Comment by Yugboon** [comment]
Multipliers I found also not working on Rampant, "unknown command sm_critchance" or another similar error is shown after shooting the multiplier.

**Comment by Dragonisser** [comment]
Nevertheless, why it doesn't work is fairly clear so im gonna close this. There will be clearer information in the future why something might not work.
